index.html

```diff
@@ -13,23 +13,25 @@
 </head>
 
 <body>
-    <header>
-        <a id="logo" href="#home"><img src="./img/Logo-Placeholder.png" alt="logo icon"></a>
-        <nav>
-            <ul class="nav-ul">
-                <li>
-                    <a href="#portfolio">Portfolio</a>
-                </li>
-                <li>
-                    <a href="#about">About</a>
-                </li>
-                <li>
-                    <a href="#contact">Contact</a>
-                </li>
-                <li><img class="nav-pic" src="./img/icon-menu.png" alt="nav menu icon"></li>
-            </ul>
-        </nav>
-    </header>
+    <nav class="nav">
+        <div class="container">
+            <div class="logo">
+                <a href="#">Temesgen</a>
+            </div>
+            <div class="main_list">
+                <ul class="nav-menu">
+                    <li><a class="nav-link" href="#portfolio">Portfolio</a></li>
+                    <li><a class="nav-link" href="#about">About</a></li>
+                    <li><a class="nav-link" href="#contact">Contact</a></li>
+                </ul>
```

Comment:
- [ ] Your nav menu is not fixed, and your mobile version has some horizontal scroll. This is causing a bad behavior and the user can experience the mobile menu make some buttons not visible(see picture below). Kindly fix that.
- [ ] In Firefox browser I experienced some overlap of the text below the mobile menu, and this is making it hard to read the buttons. Kindly fix that.

Your design:
![image](https://user-images.githubusercontent.com/61919617/175346291-654e06d0-e69c-4bd4-ad26-36d43849c8e6.png)
Firefox:
![image](https://user-images.githubusercontent.com/61919617/175346450-688c6970-3e6c-46d6-8cd4-bae8886ce2ff.png)